### ISSUE TITLE
[BUG] Terminal API error when trying to get shopper input

### ISSUE THREAD

**Issue by bluedaniel** [Adyen/adyen-node-api-library#719]
Following the documentation link below, I am getting a type error of `paymentRequest of undefined`.

Hitting the same URL with the example JSON payload above works correctly.

I'm not sure if the library covers this part of the API?

https://docs.adyen.com/point-of-sale/shopper-input/confirmation

**Comment by wboereboom** [comment]
Hi @bluedaniel,
Thanks for raising this issue!
Could you please provide me with a snippet of the call you are trying to make?

Kind regards,
Wouter
Adyen

**Comment by bluedaniel** [comment]
Using the example on the docs, this code doesn't work.

```ts
import { Client, TerminalCloudAPI } from '@adyen/api-library';

const client = new Client({apiKey: "YOUR_API_KEY", environment: "TEST"});
const terminal = new TerminalCloudAPI(client);

// Taken from https://docs.adyen.com/point-of-sale/shopper-input/confirmation
const payload = {
  "SaleToPOIRequest":{
      "MessageHeader":{
          "ProtocolVersion":"3.0",
          "MessageClass":"Device",
          "MessageCategory":"Input",
          "MessageType":"Request",
          "ServiceID":"0207111104",
          "SaleID":"POSSystemID12345",
          "POIID":"V400m-324688179"
      },
      "InputRequest":{
          "DisplayOutput":{
              "Device":"CustomerDisplay",
              "InfoQualify":"Display",
              "OutputContent":{
                  "OutputFormat":"Text",
                  "PredefinedContent":{
                      "ReferenceID":"GetConfirmation"
                  },
                  "OutputText":[
                      {
                          "Text":"Terms & Conditions"
                      },
                      {
                          "Text":"Do you accept our Terms and Conditions? \n The full text is available on our website."
                      },
                      {
                          "Text":"Decline"
                      },
                      {
                          "Text":"Agree"
                      }
                  ]
              }
          },
          "InputData":{
              "Device":"CustomerInput",
              "InfoQualify":"Input",
              "InputCommand":"GetConfirmation",
              "MaxInputTime":30
          }
      }
  }
};

const respone = await terminal.sync(payload);

```

**Comment by wboereboom** [comment]
Hi @bluedaniel ,

This is most likely caused by a bug in the previous version of our code generation tool, which did not respect API parameters starting with a capital letter.
This issue has been addressed and I'll make sure to regenerate the types and update the API accordingly. In the meantime you can confirm/fix this issue locally by updating the field names in `adyen-node-api-library/src/typings/terminal/saleToPOIRequest.ts`

Kind Regards,
Wouter
Adyen

**Comment by wboereboom** [comment]
Hi @bluedaniel,

I have reproduced your request and succesfully made a payment to a POS device using the updates made in PR #720 .
These changes have been released in version 9.0.0 of the library.

Happy coding,
Wouter
Adyen
